ouroboros-consensus/src/consensus-testlib/Test/Ouroboros/Consensus/ChainGenerator/Counting.hs

```diff
@@ -0,0 +1,341 @@
+{-# LANGUAGE ExistentialQuantification  #-}
```

Comment:
This module is a utility, much more general than `ChainGenerators`, but relied upon heavily by it.